src/test/ui/derived-errors/issue-31997.rs

```diff
@@ -10,7 +10,7 @@ fn closure<F, T>(x: F) -> Result<T, ()>
 }
 
 fn foo() -> Result<(), ()> {
-    try!(closure(|| bar(core::ptr::null_mut()))); //~ ERROR cannot find function `bar` in this scope
+    closure(|| bar(core::ptr::null_mut()))?; //~ ERROR cannot find function `bar` in this scope
```

Comment:
Change back and use `#![allow(...)]`; I don't know whether `try!` was relevant for this regression test and I'd prefer not to rock the boat.